crates/sovereign-sdk/adapters/celestia/src/celestia.rs

```diff
@@ -240,6 +244,10 @@ impl BlockHeader for CelestiaHeader {
         TmHash(self.header.hash())
     }
 
+    fn merkle_root(&self) -> Self::Hash {
+        TmHash(self.header.merkle_root())
+    }
```

Comment:
make unimplemented!() 

we don't need to update other adapters.

might be misleading for people, I don't know how celestia transaction commitments in the header work thb :)

Reply:
https://docs.rs/celestia-tendermint/latest/celestia_tendermint/block/header/struct.Header.html#structfield.data_hash